pkgs/tools/networking/connman/connman.nix

```diff
@@ -21,7 +21,6 @@
 , enableVpnc ? true
 , vpnc ? true
 , enablePolkit ? true
-, polkit ? null
```

Comment:
Still needed for polkit rules.

Reply:
What do you mean?

Reply:
`configure.ac` includes these lines:
```
POLKIT_DATADIR="`$PKG_CONFIG --variable=actiondir polkit`"
POLKIT_DATADIR=""
```
The first line accesses polkit data, the second line discards it. So for now it's indeed unused, but that may change unexpectedly. But let's do what AUR does and keep it out.

Reply:
Fair enough.

Reply:
As I said, it's okay to keep it out, like in AUR. You can remove it again.

Reply:
I think it's safer to keep it, I don't mind.

Reply:
Okay, agreed.